### PR TITLE
Dashboard: Update query group options

### PR DESCRIPTION
**Pull request by songhn233**
**What is this feature?**
- In the dashboard, tooltips have been added to some of the options in <QueryGroupOptions />, see Grafana's documentation for a description of these options
- Minutiae: The switch component is centered in the row
- Modified the placeholder for `Relative time` and `Time shift`, **the exact values may need to be discussed, my modifications are what I think is appropriate.**

**Why do we need this feature?**

 **Modified the placeholder**

`Max data points` and `Min interval` (or all fields except for the time-related field) are real values, otherwise, they are a null text description (`No limit` is displayed when `Min interval` is missing)

So two fields about time originally set to `1h` may lead to the interpretation that they take exactly `1h` (but are null) in this context, and such a placeholder setting may lead to misunderstandings, so it can be set to a null literal like `No limit`.

For `Time shift` set to `0h` because it runs as `0h` when it is null, and also implies the format of the input (math string)

Before:

image
After:

image

**tooltip and center styles**

Other fields have corresponding descriptions, while the meaning of the time-related fields is not quite easy to understand, so some of the descriptions are quoted from the documentation.

### CODE REVIEW

**Comment by songhn233** [comment]
@dprokop Hello, would you mind assisting with reviewing this change?

**Comment by songhn233** [comment]
> @dprokop Hello, would you mind assisting with reviewing this change?

@dprokop, Hey, Would you be able to take a look and provide some feedback? Thank you for the time.

File: public/app/features/query/components/QueryGroupOptions.tsx
Comment (dprokop) [inline public/app/features/query/components/QueryGroupOptions.tsx:399]:
```suggestion
            placeholder="1h"
```

Please reverti this, it's like an example shift, 0h doesn't seem like a good example value.

**Review by dprokop — CHANGES_REQUESTED** [review]
Thanks for your contibution @songhn233, and sorry for such a long wait time!

I've left some nits to fix, other than that the PR looks good!

File: public/app/features/query/components/QueryGroupOptions.tsx
Comment (songhn233) [reply public/app/features/query/components/QueryGroupOptions.tsx]: If the label here is set to `1h`, it seems possible to emphasize in the tooltip that the default value is `0h`. Otherwise, this example may be misleading to users, or they may not understand what behavior to expect when not filling in the field.

**Comment by songhn233** [comment]
@dprokop Thanks for your reply and code review. I have committed the changes according to your suggestions. 

I also added a comment. Could you please review to see if any further modifications are necessary?

**Review by axelavargas — COMMENTED** [review]
Hey @songhn233 👋🏾, thank you for your contribution. I've reviewed the PR and it looks good to me. However, there's a minor change that needs to be addressed. Also, please make sure to merge this branch with the latest main branch. :D

File: public/app/features/query/components/QueryGroupOptions.tsx
Comment (songhn233) [reply public/app/features/query/components/QueryGroupOptions.tsx]: Hey, I made some minor changes including adding examples and updating the time shift placeholder to match the other configurations.

**Comment by songhn233** [comment]
I have added some additional explanations for `interval` field, because its calculation is not exactly equal to the `Time range / max data points` described on the right.

File: public/app/features/query/components/QueryGroupOptions.tsx
Comment (axelavargas) [reply public/app/features/query/components/QueryGroupOptions.tsx:402]: 🤔 Had another look at this, and technically the main purpose of a placeholder is to provide a sample or short description of the expected value. This means if we put `0h` is because is a valid value, but this is not currently a valid value because is not going to shift the time. I think I agree with @dprokop, let's keep the `1h`

File: public/app/features/query/components/QueryGroupOptions.tsx
Comment (axelavargas) [inline public/app/features/query/components/QueryGroupOptions.tsx:378]: The same comment applies here, `No override` is not a valid sample value for this input.

**Review by axelavargas — COMMENTED** [review]
Hey @songhn233 👋🏾, thank you once again for implementing the updates. I dedicated some time to review the modifications you made concerning the placeholders and have left a few comments 😄.

File: public/app/features/query/components/QueryGroupOptions.tsx
Comment (songhn233) [reply public/app/features/query/components/QueryGroupOptions.tsx]: @axelavargas, Thanks, I also agree with this explanation.

**Review by axelavargas — APPROVED** [review]
Hey @songhn233 , thanks for the updates on the changes – they look fantastic! 🌟 From my perspective, this PR is ready. We just need @dprokop to grant approval (since he requested changes earlier, and we can't merge without it 🙂).

**Review by dprokop — APPROVED** [review]

**Pull request merged**
